esrally/mechanic/mechanic.py

```diff
@@ -33,6 +33,14 @@
 METRIC_FLUSH_INTERVAL_SECONDS = 30
 
 
+def build(cfg):
```

Comment:
This code is _very_ similar to the `download` function, but I given their entirely seperate concerns I thought it still makes sense for these to be distinct functions.